app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :user_id, dependent: :destroy
+  has_many :comments, foreign_key: :user_id, dependent: :destroy
+  has_many :likes, foreign_key: :user_id, dependent: :destroy
+
```

Comment:
- [ ] Great job so far 👍 kindly change the foreign_key from `user_id` to `author_id` and also kindly change all `user` to `author` so as to strictly follow the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram 👍 
- [ ] Kindly apply this change to all your models 💯

Reply:
Hi @Kingstalux, thank you for your feedback, I'll work on it, in this moment.